index.js

```diff
@@ -0,0 +1,114 @@
+import http from 'http';
+import fs from 'fs';
+import url from 'url'
+
+
+const requestHandler = (req, res) => {
+    const { headers, method } = req;
+    const parsedUrl = url.parse(req.url, true);
+    let body = [];
+
+    req.on('error', (err) => {
+        console.error(err.stack);
+    })
+
+
+
+    if (method === "GET" && req.url === "/") {
+        res.statusCode = 200;
+        res.setHeader('Content-Type', 'text/html');
+        res.write('<html>');
+        res.write('<head><title>Enter Message</title><head>');
+        res.write('<body><form action="/create-message" method="POST"><input type="text" name="message"><button type="submit">Submit</button></form></body>')
+        res.write('</html>');
+        return res.end();
+    }
+
+
+    if (method === "GET" && req.url.match(/^\/\?name=([^\/]+)/)) {
```

Comment:
you can keep all regex in a const

const REGEX = {
  name_regex: '',
  msg_regex: ''
}